workers/config/src/main/kotlin/ConfigWorker.kt

```diff
@@ -83,6 +83,8 @@ class ConfigWorker(
         // TODO: Currently the path to the validation script is hard-coded. It may make sense to have it configurable.
         val validationScriptExists = configManager.containsFile(resolvedJobConfigContext, VALIDATION_SCRIPT_PATH)
         val (result, validationResult) = if (validationScriptExists) {
+            logger.info("Running validation script.")
```

Comment:
Is there some else of useful information that we could log here, like the `ortRunId` or the `resolvedJobConfigContext`?

Reply:
I have added another log statement at the top, before the run id was only logged at the end.